tools/add_strelka2_fields.cwl

```diff
@@ -0,0 +1,59 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: add_strelka2_fields
+doc: >-
+  This tool will run a Python script that takes a Strelka2 tumor-normal VCF and adds
+      canonical fields like GT and AD
+requirements:
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/add-strelka2-fields:1.0.0'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${ return inputs.ram * 1000 }
+    coresMin: $(inputs.cores)
+  - class: ShellCommandRequirement
+
+baseCommand: [/usr/bin/add_strelka2_fields.py]
+
+inputs:
+  strelka2_vcf:
+    type: File
+    inputBinding:
+      position: 1
+      prefix: '--strelka2_vcf'
+    secondaryFiles: .tbi
+  tumor_name:
+    type: string
+    inputBinding:
+      position: 2
+      prefix: '--tumor_name'
+  normal_name:
+    type: string
+    inputBinding:
+      position: 3
+      prefix: '--normal_name'
+  output_basename:
+    type: string
+    inputBinding:
+      position: 4
+      prefix: '--output_basename'
+  cores:
+    type: int?
+    default: 4
+    inputBinding:
+      position: 5
+      prefix: '--cores'
```

Comment:
Does your command actually take `--cores` and `--ram` as arguments? If not, you shouldnt be binding these.

Reply:
Yeah, agree, unless the script takes those as actual args, it'll likely fail to run. Removing the prefix will allow the variables to be accessible by the requirements section without tinkering with the command args. However, the cores as an arg thing might end up having a useful home in the consensus script, if multi-threading is still being done there

Reply:
Not just the prefix. If you aren't using it in the command line you shouldn't have an inputbinding at all.